<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1546408535064" stop="1546408535064">
  <description>This is the tests collection phase. Failures are modules that failed to collect.</description>
  <name>test_collection_phase</name>
  <title>Collection phase</title>
  <labels/>
  <test-cases>
    <test-case start="1546408535064" status="broken" stop="1546408535064">
      <name>test_crm</name>
      <failure>
        <message>failed</message>
        <stack-trace>test_crm.py:100: in &lt;module&gt;
    driver.find_element_by_xpath('/html/body/div[3]/div/div/div/ul[1]/li[4]').click()
D:\SoftWare\Python3.6\lib\site-packages\selenium\webdriver\remote\webelement.py:80: in click
    self._execute(Command.CLICK_ELEMENT)
D:\SoftWare\Python3.6\lib\site-packages\selenium\webdriver\remote\webelement.py:628: in _execute
    return self._parent.execute(command, params)
D:\SoftWare\Python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py:320: in execute
    self.error_handler.check_response(response)
D:\SoftWare\Python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py:242: in check_response
    raise exception_class(message, screen, stacktrace)
E   selenium.common.exceptions.ElementNotVisibleException: Message: element not visible
E     (Session info: chrome=71.0.3578.98)
E     (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64)</stack-trace>
      </failure>
      <attachments/>
      <labels/>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
